mac/Keyman4MacIM/Keyman4MacIM/KMInputMethodAppDelegate.m

```diff
@@ -238,12 +243,27 @@ - (void)setAlwaysShowOSK:(BOOL)alwaysShowOSK {
     [userData synchronize];
 }
 
+- (void)setUseVerboseLogging:(BOOL)useVerboseLogging {
+    NSLog(@"Turning verbose logging %@", useVerboseLogging ? @"on." : @"off.");
```

Comment:
chicken or egg: Do you always want this line going to the console?

Reply:
I think so (that's why I did it that way). If you're turning it on, it's a good way to see that the user just turned it on, so you're not wondering why there was nothing in the log and all of sudden there is something.  Likewise, if they just turned it off, it allows us to see why the log suddenly stops. So in the unlikely event the user is switching it on and off, the log will show that and we won't have a really confusing log that inexplicably leaves out stuff.